newshomepages/sources/javascript/htrnews.js

```diff
@@ -0,0 +1,7 @@
+document.querySelectorAll(
+    '#gnt_mol_oy'
+  ).forEach(el => el.style.display = 'none')
+
+document.querySelectorAll('[aria-label="advertisement"]').forEach(el => el.style.display = 'none')
+
+document.querySelectorAll('#onetrust-banner-sdk').forEach(el => el.style.display = 'none')
```

Comment:
```suggestion
document.querySelectorAll(
    '#gnt_mol_oy,[aria-label="advertisement"],#onetrust-banner-sdk''
  ).forEach(el => el.style.display = 'none')
```
Here's a trick for packaging these together. What do you think?

Reply:
This definitely looks better and more concise. 

I also noticed that the last three newspapers I updated (htrnews, USAToday, and vsctar) all had the exact same javascript code. I wonder if we want to create just one file for them instead of three. I do understand that it might get a bit tricky since the code checks for the name of the file to match with the handle, but I thought I should share. 

If we do want to create one file, I am happy to update the code so it works as it should.